packages/@o3r/core/schematics/rule-factories/additional-modules/index.ts

```diff
@@ -170,8 +184,18 @@ export function updateAdditionalModules(options: { projectName: string | null },
     }
 
     const workspaceProject = getProjectFromTree(tree);
-    const envProdFilePath = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.ts');
-    if (!envProdFilePath) {
+    const envProdFilePathNg14 = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.prod.ts');
+
+    let envProdFilePath;
+    if (tree.exists(envProdFilePathNg14)) {
+      // we are in ng 14, environment prod file name is: environment.prod.ts
+      envProdFilePath = envProdFilePathNg14;
+    } else {
+      // we are in ng 14, environment prod file name is: environment.ts
+      envProdFilePath = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.ts');
+    }
```

Comment:
```suggestion
    // ng15 prod environment file name
    let envProdFilePath = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.ts');
    if (!tree.exists(envProdFilePath )) {
      // use the ng14 file name
      envProdFilePath = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.prod.ts');
    }
```

Reply:
`environment.ts` file exists in ng14, but it contains the development config (instead of prod config in ng15)
this is why we check `environment.prod.ts` to know if we are in ng14 or ng15

Reply:
oh OK, then we can do it the other way (first try the ng14 name).
My point was that we could simplify the code by not using a temporary variable and the else clause

Reply:
I could update here to use directly the prod name from ng 14 if exists. like then we simplify removing the else clause and the temp var.